.env

```diff
@@ -0,0 +1 @@
+JWT_SECRET=keep it secret, keept it safe!
```

Comment:
You want to put your .env in your `.gitignore` typically (so that people can't see your secret).